toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -483,6 +483,13 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 		generateContainerManifests(installChroot)
 	}
 
+	if config.IsKickStartBoot {
```

Comment:
I understand keeping the partitioning section to only kickstart, but this feels like a general feature we shouldn't lock behind kickstart.

Reply:
Good question. For this PR, we'll gate this feature behind kickstart only. We'll expand it towards other image build scenarios in a future work item.